__tests__/SetsSelectors.test.js

```diff
@@ -1797,62 +1797,205 @@ describe('SetsSelectors', () => {
+                        tags: ['A', 'B', 'C'],
+                        initialStartTime: '1-2-18'
+                    },
+                    b: {
+                        setID: 'b',
+                        exercise: 'Squat',
+                        weight: 200,
+                        metric: 'lbs',
+                        reps: [{
+                            isValid: true,
+                            removed: false,
+                            data: [-3456, 37, 1.943368, 388, 38, 28, 1, 18]
+                        }],
+                        tags: ['A', 'B', 'C'],
+                        initialStartTime: '2018-01-03T04:06:12.640Z'
+                    },
+                    c: {
+                        setID: 'c',
+                        exercise: 'Bench',
+                        weight: 100,
+                        metric: 'lbs',
+                        reps: [{
+                            isValid: true,
+                            removed: false,
+                            data: [-3456, 37, 1.733368, 288, 28, 18, 1, 8]
+                        }],
+                        tags: ['A', 'E', 'B', 'C'],
+                        initialStartTime: '2018-01-03T04:06:12.640Z'
+                    },
+                    d: {
+                        setID: 'd',
+                        exercise: 'Bench',
+                        weight: 200,
+                        metric: 'lbs',
+                        reps: [{
+                            isValid: true,
+                            removed: false,
+                            data: [-3456, 37, 1.833368, 188, 18, 8, 1, 4]
+                        }],
+                        tags: ['A', 'B', 'F', 'C'],
+                        initialStartTime: '2018-01-03T04:06:12.640Z'
+                    },
+                }
+            },
+            analysis: {
+                e1RMDaysRange: 7,
+                tagsToInclude: ['A', 'B', 'C'],
+                tagsToExclude: ['E'],
             }
+        };
 
-            const expected = [
-                {label: "Box Squat w/ Bands", value: "Box Squat w/ Bands"}, 
-                {label: "Good Mornings w/ Bands & Chains in Belt Squat", value: "Good Mornings w/ Bands & Chains in Belt Squat"}, 
-                {label: "Zercher Squats", value: "Zercher Squats"}, 
-                {label: "Close Grip Bench Press", value: "Close Grip Bench Press"}, 
-                {label: "Reverse Band Bench", value: "Reverse Band Bench"}, 
-                {label: "SSB Squat", value: "SSB Squat"}, 
-                {label: "Deadlift w/ Chains", value: "Deadlift w/ Chains"}]
+        test('get all tag suggestions to include for an exercise', () => {
+            const expected = [];
 
-            const result = sut.generateExerciseItems(state);
+            const result = sut.getTagsToIncludeSuggestions(state, 'Bench');
 
-            expect(result).toEqual(expected);
+            expect(result).toEqual(["A", "B", "C", "F", "X"]);
```

Comment:
- [ ] Wait, shouldn't getTagsToIncludeSuggestions actually only show you things that aren't already included? If you've already included A, you don't want to see A again